### PR TITLE
feat(jobdsl): jobdsl tidb merged pipeline

### PR DESCRIPTION
**Pull request by purelind**
* make tidb merge pipeline jobdsl for easy maintenance and updates

### CODE REVIEW

File: jenkins/jobs/ci/tidb/ghpr_common_test.groovy [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                    triggerPhrase('.*\/(run(-integration-tests|-common-test).*)')
+                    onlyTriggerPhrase(true)
+                    skipBuildPhrase(".*skip-ci.*")
+                    buildDescTemplate('PR #$pullId: $abbrTitle\n$url')
+                    whitelist('')
+                    orgslist('pingcap')
+                    whiteListTargetBranches {
+                        ghprbBranch { branch('master') }
+                        ghprbBranch { branch('^(release-)?5\\.[0-4]\\d*(\\.\\d+)?(\\-.*)?$') }
+                        ghprbBranch { branch('^(release-)?6\\.[2-9]\\d*(\\.\\d+)?(\\-.*)?$') }
```
Comment (wuhuizuo) [inline jenkins/jobs/ci/tidb/ghpr_common_test.groovy:29]: dose it need to support feature branches?

Comment (wuhuizuo) [reply jenkins/jobs/ci/tidb/ghpr_common_test.groovy:29]: /lgtm rest

**Review by wuhuizuo — APPROVED** [review]

File: jenkins/jobs/ci/tidb/ghpr_common_test.groovy (hunk 1 above)
Comment (purelind) [reply jenkins/jobs/ci/tidb/ghpr_common_test.groovy:29]: Consistent with existing behavior

**Pull request merged**